requirements.txt

```diff
@@ -5,4 +5,5 @@ pyyaml>=3.13
 fastcore>=0.1.17
 opencv-python>=4.2.0
 pytorch-lightning>=0.7.6
-torchvision==0.6.0
\ No newline at end of file
+torchvision==0.6.0
+timm
```

Comment:
I'm still not sure this should be a requirement, let's wait a bit and see

Reply:
Removed.